### PR TITLE
Unit tests for PLIER data model and symbol mapping script

### PR DESCRIPTION
**Pull request by jjc2718**
I plan to add more tests going forward - this is just a starting point for covering some of the edge cases that have come up during development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,3 +37,15 @@ install_github('wgmao/PLIER')
...
 without errors.
+
+## Running tests
+
+Running the tests requires the `pytest` module (included in the specified
+Conda environment). Once this module is installed, you can run the tests
+by executing the command
+
+```shell
+python -m pytest tests
```
Comment (zietzm) [inline README.md]: Not sure if there is a difference, but you can also run tests with `pytest tests/`

File: tests/test_symbol_to_entrez_id.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import math
+import unittest
```
Comment (zietzm) [inline tests/test_symbol_to_entrez_id.py]: Tests all seem good, but is there a reason you are using `unittest` over `pytest`?

File: tests/test_symbol_to_entrez_id.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        assert len(list(gene_map.keys())) == len(symbols)
+        assert 'N/A' not in list(gene_map.values())
+
+    def test_nan_entrez_id(self):
+        """Test the case where a symbol is found in the DB, but without an ID.
+
+        This can happen for either the symbol proper or one of its aliases,
+        so we need to test both cases.
+        """
+        # BRAF is a well-studied gene which has an exact match
```
Comment (zietzm) [inline tests/test_symbol_to_entrez_id.py]: Very nice, don't often enough see description of the test case like this

File: tests/test_symbol_to_entrez_id.py (hunk 2 above)
Comment (jjc2718) [reply tests/test_symbol_to_entrez_id.py]: No good reason - rewrote tests in `pytest` style and got rid of `unittest` dependency.
